tests/Unit/Parallel/Test_PupStlCpp11.cpp

```diff
@@ -52,44 +52,44 @@ SPECTRE_TEST_CASE("Unit.Serialization.unordered_map", "[Serialization][Unit]") {
   std::unordered_map<std::string, double> um;
   um["aaa"] = 1.589;
   um["bbb"] = -10.7392;
-  CHECK(um == serialize_and_deserialize(um));
+  test_serialization(um);
 }
 
 SPECTRE_TEST_CASE("Unit.Serialization.enum", "[Serialization][Unit]") {
-  eDummyEnum test1 = eDummyEnum::test2;
-  CHECK(test1 == serialize_and_deserialize(test1));
+  eDummyEnum e = eDummyEnum::test2;
+  test_serialization(e);
 }
 
-/// [example_serialize_copyable]
+/// [example_serialize_comparable]
```

Comment:
why did this change?

Reply:
because the test function requires the class to be comparable, not copyable